packages/web-app-files/src/composables/actions/spaces/useSpaceActionsUploadImage.ts

```diff
@@ -56,6 +57,38 @@ export const useSpaceActionsUploadImage = ({
       extraHeaders['X-OC-Mtime'] = '' + file.lastModified / 1000
     }
 
+    try {
+      await clientService.webdav.getFileInfo(selectedSpace, { path: '.space' })
+    } catch (_) {
+      await clientService.webdav.createFolder(selectedSpace, { path: '.space' })
+      await clientService.webdav.putFileContents(selectedSpace, {
+        path: '.space/readme.md',
+        content: $gettext('Here you can add a description for this Space.')
+      })
+      const file = await clientService.webdav.getFileInfo(selectedSpace, {
+        path: '.space/readme.md'
```

Comment:
I wonder if we can outsource the code to a helper func, like: InitializeSpaceStructure and call it anywhere we need it, for example in the space overview? (while createing a space)

Reply:
@JanAckermann Check last commit please. Do you mean something like that?